pgctl/config.py

```diff
@@ -116,6 +116,10 @@ def from_app(self, path='.'):
             for parentdir in reversed(tuple(search_parent_directories(path)))
         )
 
+    def from_config_arg(self, args):
+        if hasattr(args, 'config') and args.config is not None:
```

Comment:
this `hasattr` should always be true -- argparse will make it `None` if it isn't present

Reply:
Dummy, which is the default for args passed to Config, does not have this attribute. The test `tests/spec/config.py::DescribeCombined::it_can_be_run_via_python_m` in particular fails, because it doesn't pass command line arguments into Config.combine. This seemed like the least insane solution to this problem. I'm very open to suggestions though.

Reply:
I realized I could just have Dummy have config as an attribute and that fixes all the actual issues.